src/App.js

```diff
@@ -1,9 +1,9 @@
-/* eslint-disable */
+/* eslint-disable no-useless-constructor */
+/* eslint-disable react/prefer-stateless-function */
 import './App.css';
 import React from 'react';
 import Calculator from './components/Calculator';
 
-
 class App extends React.Component {
```

Comment:
- [ ] OPTIONAL - Rather than disabling the `eslint react/prefer-stateless-function` consider extending from `React.PureComponent` instead to fix it.

Reply:
ok thanks very much for the advice. I never knew how to fix it